cli/explain/explain.go

```diff
@@ -76,6 +76,7 @@ var (
 	oldLog     = explainCmd.String("old", "", "Path to a compact execution log or invocation ID of a build to consider as the baseline for the diff.")
 	newLog     = explainCmd.String("new", "", "Path to a compact execution log or invocation ID of a build to compare against the baseline.")
 	verbose    = explainCmd.Bool("verbose", false, "Print more detailed execution information.")
+	apiTarget  = explainCmd.String("target", "", "The API target to use for fetching logs instead of the last --bes_backend.")
```

Comment:
This may be coming in a future PR but should this be set in the FE?

Reply:
Yes, this will have to wait for a release.